src/main/java/com/store/bookstore/repository/role/RoleRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.store.bookstore.repository.role;
+
+import com.store.bookstore.models.Role;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface RoleRepository extends JpaRepository<Role, Long> {
+    @Query(value = "SELECT * FROM roles WHERE role = :roleName", nativeQuery = true)
+    Optional<Role> findByName(Role.RoleName roleName);
```

Comment:
I guess you can change method name a little bit and remove `@Query`
```suggestion
    Optional<Role> findByRoleName(Role.RoleName roleName);
```